core/scripts/generate_sdk_index.sh

```diff
@@ -5,7 +5,7 @@
 cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
 
 # Array of directories to process
-declare -a dirs=("../sdk/src")
+declare -a dirs=("../../packages/sdk/src")
```

Comment:
this is an array, we can just add more folders here.

Reply:
thank you!